modules/book.js

```diff
@@ -0,0 +1,64 @@
+export default class Book {
+  constructor(title, author) {
+    this.title = title;
+    this.author = author;
+  }
+
+  // getbook  method
+  static getBooks() {
+    let books;
+    if (localStorage.getItem('books') === null) {
+      books = [];
+    } else {
+      books = JSON.parse(localStorage.getItem('books'));
+    }
+    // console.log(books);
```

Comment:
Good job using ES6 syntax. However, please remove unused commented code from the codebase it is not the best practice to leave commented code.